src/Sylius/Bundle/InstallerBundle/Behat/CliContext.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\InstallerBundle\Behat;
+
+use Sylius\Bundle\InstallerBundle\Command\SetupCommand;
+use Sylius\Bundle\ResourceBundle\Behat\DefaultContext;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Helper\DialogHelper;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+
+
+/**
+ * @author Magdalena Banasiak <magdalena.banasiak@lakion.com>
+ */
+class CliContext extends DefaultContext
+{
+    /**
+     * @var Application
+     */
+    private $application;
+
+    /**
+     * @var CommandTester
+     */
+    private $tester;
+
+    /**
+     * @var DialogHelper
+     */
+    private $dialog;
+
+    /**
+     * @var SetupCommand
+     */
+    private $command;
+
+    /**
+     * @var array
+     */
+    private $inputChoices = array(
+        'currency' => '',
+        'name' => ' Name',
+        'surname' => ' Surname',
+        'e-mail' => ' test@email.com',
+        'password' => ' pswd',
+        'confirmation' => ' pswd'
+    );
+
+    /**
+     * @When /^I run a command "([^"]+)"$/
+     */
+    public function iRunACommand($name, $parameters = array())
```

Comment:
How do we receive $parameters for this function? If I am right, the regexp will match only one word, the command name.